models/aeif_cond_exp_implicit.nestml

```diff
@@ -115,9 +115,9 @@ neuron aeif_cond_exp_implicit:
       emit_spike()
     end
 
-    g_ex += spikeExc.get_sum()
-    g_in += spikeInh.get_sum()
-    I_stim = currents.get_sum()
+    g_ex += spikeExc * 1 nS
+    g_in += spikeInh * 1 nS
+    I_stim = currents
```

Comment:
currents are per construction always `pA`, since it models an external current which has by definition `pA` as its dimension. In contrast to current buffers, spike buffers may be either current-based or conductance based. Since it is not always possible (or it is not transparent for the modeler) we decided to make spike buffers unitless. Thus, the modeler is forced to state the desired type of the input by multiplying it with a corret unit literal (1pA in the case of current based models, 1nS in the case of the conductance based models).